### PR TITLE
onmidimessage event change

### PR DESCRIPTION
**Pull request by Rumyra**
#### Summary

`onmidimessage` should be `midimessage_event`

#### Motivation

Updating midi docs, so good chance to change the event pages - this pr tackles `onmidimessage`. This is my first go so am drafting and pinging @teoli2003 @Elchi3 or @queengooborg for review 🙏  (Also not I haven't checked bcd yet - I wasn't sure what comes first)

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

Part of [#12792](https://github.com/mdn/content/issues/12792)

### CODE REVIEW

File: files/en-us/web/api/midiinput/midimessage_event/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Event type
+
+An {{domxref("MIDIMessageEvent")}}. Inherits from {{domxref("Event")}}.
+
+{{InheritanceDiagram("MIDIMessageEvent")}}
+
+## Event properties
+
+{{page("/en-US/docs/Web/API/MIDIMessageEvent", "Properties")}}
```
Comment (teoli2003) [inline files/en-us/web/api/midiinput/midimessage_event/index.md:34]: We are getting rid of {{page}} on MDN, plus copy and adapt the section.

File: files/en-us/web/api/midiinput/midimessage_event/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Examples
+
+In the following example `midimessage` events are listened for on all input ports. When a message is received the {{domxref("MIDIMessageEvent.data")}} property is printed to the console.
+
+```js
+inputs.forEach((input) => {
```
Comment (teoli2003) [inline files/en-us/web/api/midiinput/midimessage_event/index.md:41]:
```suggestion
inputs.forEach(input => {
```

File: files/en-us/web/api/midiinput/index.md [hunk 3]
```diff
@@ -21,23 +21,23 @@ The **`MIDIInput`** interface of the [Web MIDI API](/en-US/docs/Web/API/Web_MIDI
 
 _This interface doesn't implement any specific properties, but inherits properties from {{domxref("MIDIPort")}}._
 
-### Event handlers
+### Events
```
Comment (teoli2003) [inline files/en-us/web/api/midiinput/index.md:24]: This section must be after the _Methods_ section, according to the template.

File: files/en-us/web/api/midiinput/index.md [hunk 4]
```diff
@@ -21,23 +21,23 @@ The **`MIDIInput`** interface of the [Web MIDI API](/en-US/docs/Web/API/Web_MIDI
...
 
-- {{domxref("MIDIInput.onmidimessage")}}
-  - : When the current port receives a MIDI message it triggers a call to this event handler.
+- {{domxref("MIDIInput.midimessage_event")}}
```
Comment (teoli2003) [inline files/en-us/web/api/midiinput/index.md:26]:
```suggestion
- {{domxref("MIDIInput.midimessage_event", "midimessage"}}
```

File: files/en-us/web/api/midiinput/index.md [hunk 5]
```diff
@@ -21,23 +21,23 @@ The **`MIDIInput`** interface of the [Web MIDI API](/en-US/docs/Web/API/Web_MIDI
...
+  - : When the current port receives a MIDI message it triggers a `midimessage` event.
```
Comment (teoli2003) [inline files/en-us/web/api/midiinput/index.md:27]:
```suggestion
  - : Fired when the current port receives a MIDI message.
```

File: files/en-us/web/api/midimessageevent/data/index.md [hunk 6]
```diff
@@ -25,11 +25,11 @@ A {{jsxref("Uint8Array")}}.
 
 ## Examples
 
-In the following example {{domxref("MIDIInput.onmidimessage")}} events are listened for on all input ports. When a message is received the value of `data` is printed to the console.
+In the following example {{domxref("MIDIInput.midimessage_event","midimessage events")}} are listened for on all input ports. When a message is received the value of `data` is printed to the console.
```
Comment (teoli2003) [inline files/en-us/web/api/midimessageevent/data/index.md:28]:
```suggestion
In the following example {{domxref("MIDIInput.midimessage_event", "midimessage")}} events are listened for on all input ports. When a message is received the value of `data` is printed to the console.
```

File: files/en-us/web/api/midimessageevent/data/index.md [hunk 7]
```diff
@@ -25,11 +25,11 @@ A {{jsxref("Uint8Array")}}.
...
 
 ```js
 inputs.forEach((input) => {
-  input.onmidimessage = function(message) {
+  input.onmidimessage = (message) => {
```
Comment (teoli2003) [inline files/en-us/web/api/midimessageevent/data/index.md:32]:
```suggestion
  input.onmidimessage = message => {
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
This is really good. A few nit and a couple of things to move around.

**Comment by Rumyra** [comment]
I think I've address everything @teoli2003 I just need to take a look at bcd next 👍

File: files/en-us/web/api/midiinput/index.md [hunk 8]
```diff
@@ -21,23 +21,23 @@ The **`MIDIInput`** interface of the [Web MIDI API](/en-US/docs/Web/API/Web_MIDI
...
-- {{domxref("MIDIInput.onmidimessage")}}
-  - : When the current port receives a MIDI message it triggers a call to this event handler.
-
 ## Methods
 
 _This interface doesn't implement any specific methods, but inherits methods from {{domxref("MIDIPort")}}._
 
+### Events
+
+- {{domxref("MIDIInput.midimessage_event", "midimessage"}}
```
Comment (teoli2003) [inline files/en-us/web/api/midiinput/index.md:30]: This is the test failure.

```suggestion
- {{domxref("MIDIInput.midimessage_event", "midimessage")}}
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/midiinput/index.md (hunk 8 above)
Comment (Rumyra) [reply files/en-us/web/api/midiinput/index.md]: Ooo good catch!

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**